### ISSUE TITLE
Modernize the League of Women Voters guide 

### ISSUE THREAD

**Issue by mariochampion** [open-austin/project-ideas#84]
### What problem are we trying to solve?

Over 80,000 people downloaded the 2016 LWV austin election guide pdf from their site, not counting those who grabbed it from both the Travis County elections and Austin American Statesman sites (and maybe others). It is a widely used document, but it is a **PDF**, and so not searchable, not API-able (or keyword/tag bot-friendly), not mobile-friendly, and, as they know, ugly as all heck.

There is an opportunity to modernize and re-structure how they collect, store, update, disseminate this highly-sought information. From straight up ui/ux concerns across a range of presentations (web to tablet to mobile) to datastore and data structure elements to allow search, interest tagging for alerts, geo-tagging for relevance, and other meta-information as well -- and much more. 

They might still end up with a PDF for folks who desire that, but they wouldnt have JUST a pdf (with spaces in the URL ; )

### Who will benefit (directly and indirectly) from this project?

**directly:** LWV and their supporters, "customers", and, more broadly, engaged citizens who like to be informed when they vote.

**indirectly:** OpenAustin can expand it's Partnerships roster, so to speak, reaching a wide, already existing audience, and perhaps nudge more tech folks into civic engagement. Thinking longer term, after a successful Austin pilot, it could be an open role-model for [other LWV chapters](http://lwv.org/get-involved/local-leagues), of which there are 200 or so.

### Links to any research/data available/articles

overview: [http://lwvaustin.org/voter-guide/](http://lwvaustin.org/voter-guide/)
actual guide: [English PDF](http://lwvaustin.org/votersguide/Voters%20Guide%20Nov%202016%20Update%20-%20English.pdf) and  [Spanish PDF](http://lwvaustin.org/votersguide/Voters%20Guide%20Nov%202016%20Update%20-%20Spanish.pdf)

### What are the next steps (validation, research, coding, design)? 

OK, so some good news -- i happen to be friends and a fellow Urban Transportation Commissioner with Cinde Weatherby, the LWV Austin president. I ran this idea by her after it came up at the Dec Open Austin meetup and she responded with an enthusiastic "yes."

### What help is needed at this time?

We have  a little time, as i write this 14 dec 2016 and there are no general elections for a while, to wrangle some roles: 
* product/project lead
* design lead
  * front-end, ui/ux/css/sass/etc
* back-end lead
  * data store/structures, APIs, middleware, etc
and probably others as this progresses.

I have [loads of experience](http://mariochampion.com) in all the above roles, but thats not the best way to pursue a collaborative project, imho. I ve moved away from writing as much code as used to write (spent the better part of the last 10+ years doing LAMP/ubuntu/nginx/JS/ajaxy-jsonp things (and a lot more) and ui/ux/css/creativecloud design bits, too.

I d be happy to project lead -- collect requirements/understand their current workflow, figure out hosting/permissions per organization, structure new data tools and params, set up milestones, etc.

Then wireframe/prototyping along with whomever wants to fill front-end and back-end design and data roles to actually build it. 

ALL roles can be filled from the beginning *(that would be ideal)* but the project could get started without just me and cinde.

comments, ideas, offers to jump in? please do!

**Comment by VictoriaODell** [comment]
This is a great idea @mariochampion! Are you in the slack channel too? That might be a good place to recruit some people interested in getting involved.

**Comment by bynines** [comment]
I'd love to help as design lead, if you're still in need.

**Comment by bripenney** [comment]
Also would love to help with design!

**Comment by mariochampion** [comment]
Hey Joshua,

Thanks! Just posted this today, so I ll certainly hit you up when it gets rolling. 

Talk soon,
Mario

---
Pardon the brevity, sent from my secret hideout.

**Comment by mariochampion** [comment]
Hey bripenny, (ha, Autocorrect wants your name to be "bro penny")

Thanks, I just posted this today so I ll email ya when it gets rolling and we can divvy up roles and figure out how to collaborate. 

Talk soon,
Mario

---
Pardon the brevity, sent from my secret hideout.

**Comment by mariochampion** [comment]
Hey, 

Thanks -- the idea came from someone at meetup, don't recall exactly who mentioned it. Prolly Matteo will remember. (I m on phone so I can't @mention him)

I ll jump on the slack channel soon. I need to coordinate with Cinde first to get a better estimate on her timelines. 

Talk soon,
Mario

---
Pardon the brevity, sent from my secret hideout.

**Comment by mateoclarke** [comment]
I think it was @benguhin who threw out the observation that LoWV has excellent, well respected candidate information but abysmal design.

**Comment by benguhin** [comment]
This is true! It would be awesome to work with them directly on how to format their content so it's more accessible and available (e.g., content APIs)

**Comment by mateoclarke** [comment]
@mariochampion, any update from Cinde on the timeline for LWV?

**Comment by mariochampion** [comment]
hey @mateoclarke and all, 
just heard back again from Cinde (LWV pres) and she is still "totally under water" with the surge of new members and efforts. They are only putting out a "very slim guide" for the May elections (which dont affect austin), so she thinks by March we can get together to talk about their process, changes that would be viable.

I ll let people on this thread know and we ll see where we can guide a new guide ; )

**Comment by mateoclarke** [comment]
Cool thanks for the update!

**Comment by thedeadline** [comment]
Another person willing to help you with design, since the labels are still there. You can contact me here: hello@isabelleg.me

**Comment by texasdmh** [comment]
I just noticed this thread.  I can make a good case that the LWV leaving my
race out of the printed guide this election season cost me the race; or
came close to it.  If I can help in any way please let me know.

=====================================================
David Holmes
http://votedavidholmes.com/
512.524.9791

David's Facebook <http://www.facebook.com/intrepidas>
David's LinkedIn <http://www.linkedin.com/in/intrepidas>

**Comment by werdnanoslen** [comment]
@mariochampion what's the status on this project? Can I help you move this forward?

**Comment by a-rabbit** [comment]
Hi  I am interested in helping with this project.

**Comment by civictechnologyfoundation** [comment]
Okay folks,
I'm soooo glad to see the enthusiasm here.  Where have you folks been?!?!?
Okay ->
We're about four years ahead of you all on this one.  
First, download The Voting App - Represent!SA [iOS ](http://bit.ly/repsa-ios)and [Android](http://bit.ly/repsa-android) apps and check out what we did to display the LWV Voters Guides (amongst 3 other org voters guides including our own Candidate Video Voters Guide).  I recommend you find a San Antonio address to plug into the app to test it out.  We're pretty happy with it and so is San Antonio.
We had released a version of this app across Texas in the 2016 General Election and are still attempting to go nationwide.  We dialed back the coverage for a more concentrated test and are currently in the middle of a big pilot campaign in San Antonio's Municipal Election and Runoff. Hence the current intentional isolation of the data to San Antonio.
We actually already created an API that ingests the exported LWV Vote411 Voters Guide CSV data and makes it parsable.   We originally created this tool as ThinkVoting and have since made our backend technology OpenSource via our nonprofit,[ Civic Technology Foundation](http://civictechfoundation.org)  We want a community of programmers to beef this thing up.  In fact we have much to work on to get this where we want it to be before the November elections.  Reach out to me.
Things you should know before getting deep into this:
The local LWVs use an online platform to gather and display their Voters Guide information online.  This guide is called [Vote411](http://vote411.org) The platform is developed by a third party org, but they recently dissolved and turned their technology over to the US LWV.  This third party org claimed to have an API for this system already.  Our tests found it to be somewhat incomplete and a bit clunky, but at least it's a start.  There's more to the story if you're interested.
FYI, we have openSourced our backend technology that currently takes the exported CSV files from the eThePeople Vote411.  We are hoping to expand this functionality to be able to fully handle the eThePeople API and fill in the holes they currently have.  Who wants to jump on board to make this even more badass?  
BTW, there's more to the story if you're interested.  Reach out to me via email.
jsantori@civictechfoundation.org

**Comment by werdnanoslen** [comment]
That's great @civictechnologyfoundation! If you're ever in town, I'd like to hear more and maybe show us how we could fork/contribute to this for Austin.

**Comment by civictechnologyfoundation** [comment]
We are in Austin.  Let's  meet for coffee.  Lots to talk about.  Hit me up at the email in the above comment.

**Comment by werdnanoslen** [comment]
@a-rabbit @mariochampion are y'all available tomorrow to chat over some beer or coffee? I'd like to see how we can continue on this project.

**Comment by a-rabbit** [comment]
Sorry, my in laws rescheduled and are coming up for the day tomorrow. I am
free during the week, if you want to meet up then.

Amy

**Comment by werdnanoslen** [comment]
Sketch City (Houston) had some similar projects recently:
https://github.com/sketch-city/project-ideas/issues/30
https://github.com/sketch-city/project-ideas/issues/60
https://github.com/sketch-city/project-ideas/issues/66
https://github.com/sketch-city/project-ideas/issues/68
https://github.com/sketch-city/project-ideas/issues/113

**Comment by gazugafan** [comment]
Anyone make any progress on this? I've been reaching out to e.thepeople and Vote411 to try to get any sort of API access, but haven't been able to get very far yet. I'm in Austin if anyone wants to grab a coffee and talk! Would love to hear more about your open-source Vote411 parser, @civictechnologyfoundation!

**Comment by mscarey** [comment]
See also #140, BallotAPI
